perf-runner/src/github-api.mjs

```diff
@@ -0,0 +1,41 @@
+import { config } from './config.mjs';
+
+/**
+ * Post a comment to a GitHub PR
+ */
+export async function postComment(message) {
```

Comment:
If we are using GHA we can just rely on their PR checks instead of making comments right? They even have an html reports: https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-commands#adding-a-job-summary